noxfile.py

```diff
@@ -39,6 +39,13 @@ def test_pythonpath(session):
     run_tests(session, env={'PYTHONPATH': 'src'})
 
 
+@nox.session(python='3.8')
+def docs(session):
+    session.install('.', '-r', 'docs/requirements.txt')
+    output = session.create_tmp()
+    session.run('python', '-m', 'sphinx', '-n', '-W', 'docs/source', output)
```

Comment:
```python
os.path.join('docs', 'source')
```

Reply:
Ah! And same for `docs/requirements.txt`.

Reply:
(This is my first time actually mucking with nox, I couldn't tell if it makes chdir guarantees -- good catch sounds like, will change)

Reply:
Oh wait no, sorry, that's for os.sep reasons? I saw/thought about that but I saw another place in the noxfile that was just using '/' -- should I fix here too: https://github.com/FFY00/python-build/blob/master/noxfile.py#L16-L17 ?

Reply:
Yes. Please fix if you can, but do it in a separate PR to not mix the changes :grin:

Reply:
Sorry, not PR, different commit. I just woke up a few minutes ago :sweat_smile:

Reply:
Fixed!